client/components/FormattingBar/controlComponents/ControlsReference.tsx

```diff
@@ -53,4 +57,44 @@ const ControlsReference = (props: ControlsReferenceProps) => {
 	);
 };
 
+type ControlsReferencePopoverContentProps = {
+	pubData: any;
+	small?: boolean;
+	dark?: boolean;
+};
+
+export const ControlsReferenceSettingsLink = (props: ControlsReferencePopoverContentProps) => {
+	const link = (
+		<a
+			href={getDashUrl({
+				pubSlug: props.pubData.slug,
+				mode: 'settings',
+				section: 'block-labels',
+			})}
+		>
+			Pub Settings
+		</a>
+	);
+
+	return (
+		<span className={classNames('controls-reference-settings-link', props.dark && 'dark')}>
+			{props.small ? (
+				<>Enable block labeling in {link}</>
```

Comment:
A bit late to this one but I wonder if we should just call this feature "item labeling"? I think the word "block" may not do much here for someone unfamiliar with the internals of our editor schema.

Reply:
I like "item labeling" a lot more. I'll go ahead and make the change now with the assumption we'll get more feedback from the team before this gets merged into master.